### PR TITLE
feat: add sagemaker finetuning notebook

### PR DESCRIPTION
**Pull request by mike-meow**
Sagemaker finetuning notebook from https://github.com/cohere-ai/cohere-aws/blob/main/notebooks/sagemaker/Run%20command%20R%20finetuning.ipynb

### CODE REVIEW

**Comment by kyle-cohere** [comment]
Comments:

- Please replace the title with a level 2 header (`##` rather than `#`)
- The example training data doesn't look great—are you sure you want it like that? It would probably look better if you formatted it with 2 spaces indent and labelled the block as json (see below). Also you'll need to replace the single quotes with double quotes if you want it to be valid JSON
- If you want to format the argument values tables correctly, you'll need to put them on their own line

Better example data:
```json
{
  "messages": [
    {
      "role": "System",
      "content": "You are a chatbot trained to answer to my every question."
    },
    {
      "role": "User",
      "content": "Hello"
    },
    {
      "role": "Chatbot",
      "content": "Greetings! How can I help you?"
    },
    {
      "role": "User",
      "content": "What makes a good running route?"
    },
    {
      "role": "Chatbot",
      "content": "A sidewalk-lined road is ideal so that you\u2019re up and off the road away from vehicular traffic."
    }
  ]
}
```

File: notebooks/finetuning/Command R finetuning sagemaker.ipynb [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cohere\n",
```
Comment (CoderHam) [inline notebooks/finetuning/Command R finetuning sagemaker.ipynb:63]:
```suggestion
    "!pip install cohere>=0.5.11\n",
```

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: notebooks/finetuning/Command R finetuning sagemaker.ipynb (hunk 1 above)
Comment (youran-qi) [reply notebooks/finetuning/Command R finetuning sagemaker.ipynb:63]: Should be `!pip install cohere>=5.11.0`?

Comment (CoderHam) [reply notebooks/finetuning/Command R finetuning sagemaker.ipynb:63]: Yes

**Review by CoderHam — APPROVED** [review]

**Review by youran-qi — APPROVED** [review]

**Pull request merged**